File: go/solutions/1672-richest-customer-wealth.go
package solutions

func MaximumWealth(accounts [][]int) int {
	ans := 0
	for i := 0; i < len(accounts); i++ {
		sum := 0
		for j := 0; j < len(accounts[i]); j++ {
			sum += accounts[i][j]
		}
		ans = max(ans, sum)
	}

	return ans
}

File: js/1672-richest-customer-wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function (accounts) {
  return Math.max(
    ...accounts.map((arr) => arr.reduce((acc, curr) => acc + curr, 0))
  );
};

module.exports = { maximumWealth };
